docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -485,6 +527,37 @@ Extract the year quarter the date/datetime falls in.
 include-tagged::{sql-specs}/docs.csv-spec[quarter]
 --------------------------------------------------
 
+[[sql-functions-today]]
+==== `TODAY`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+TODAY()
+--------------------------------------------------
+
+*Input*: _none_
+
+*Output*: date
+
+.Description:
+
+This function offers the same functionality as <<sql-functions-current-date,CURRENT_DATE()>> function: returns
+the date when the current query reached the server. This method always returns the same value within a query.
```

Comment:
Here, the same comment about the clarity of "same value within a query"... as above.